Makefile

```diff
@@ -16,6 +16,8 @@ all: $(addprefix build-,$(ARCH))
 # Image registry for build/push image targets
 export IMAGE_REGISTRY ?= ghcr.io/external-secrets/external-secrets
 
+#Valid licenses for check-licenses 
+LICENSES ?= Apache-2.0|MIT|BSD-3-Clause|ISC|MPL-2.0|BSD-2-Clause|Unknown
```

Comment:
If we remove Unknown, we get the current behavior where `akeyless.go` SDK has no licenses (and it is an error). Should we keep it like that and track akeyless licenses elsewhere?